docs/en/setup/plugins/server_grpc-server.md

```diff
@@ -15,6 +15,12 @@ max_concurrent_streams: 32
 tls_cert_file: ""
 # The TLS key file path.
 tls_key_file: ""
+# To Accept Connection Limiter when reach the resource
+accept_limit:
+  # The max CPU utilization limit
+  cpu_utilization: 90
```

Comment:
I think `75%` is a more common default setting for CPU load. And have you verified this would be over 100% in multiple cores?

Reply:
It works too. I have combined all the CPU information into one utilization.